Sources/KituraContracts/Contracts.swift

```diff
@@ -295,6 +295,12 @@ public protocol QueryParams: Codable {}
 
 /**
  An error representing a failure to create an `Identifier`.
+
+### Usage Example: ###
+ ````swift
+ // An `IdentifierError.invalidValue` may be thrown if the given string cannot be converted to an integer when using an `Identifier`.
```

Comment:
Please can you take this out of a code comment and write it in text above the code sample so that it won't require scrolling across the screen to read.